tests/testthat/test-parser.R

```diff
@@ -47,6 +47,7 @@ test_that("df2vw correctly parses data", {
     )
     
     # Regular labels
+    print("Regular labels")
```

Comment:
Please use `cat()` instead of `print()`. Print is meant to show the contents inside of data structures whereas `cat()` treats the atomic types literally. c.f.

```r
print("foo")
# [1] "foo"
cat("foo\n")
# foo
```

c.f. https://stackoverflow.com/questions/36699272/why-is-message-a-better-choice-than-print-in-r-for-writing-a-package/36700294#36700294